timemachine/cpp/src/kernels/k_rotations.cu

```diff
@@ -88,16 +88,17 @@ void __global__ k_rotate_coordinates(
 
 template <typename RealType, bool SCALE>
 void __global__ k_rotate_and_translate_mols(
-    const int num_samples,
+    const int batch_size,
     const double *__restrict__ coords,         // [N, 3]
     const double *__restrict__ box,            // [3, 3]
-    const int *__restrict__ samples,           // [num_samples]
-    const int *__restrict__ mol_offsets,       // [max_sample_idx+1]
-    const RealType *__restrict__ quaternions,  // [num_samples, 4]
-    const RealType *__restrict__ translations, // [num_samples, 3]
-    double *__restrict__ coords_out) {
+    const int *__restrict__ samples,           // [batch_size]
+    const int *__restrict__ mol_offsets,       // [batch_size + 1]
+    const RealType *__restrict__ quaternions,  // [batch_size, 4]
+    const RealType *__restrict__ translations, // [batch_size, 3]
+    double *__restrict__ coords_out            // [batch_size, num_atoms, 3]
+) {
 
-    int sample_idx = blockIdx.x * blockDim.x + threadIdx.x;
+    int batch_idx = blockIdx.x * blockDim.x + threadIdx.x;
```

Comment:
`batch_idx` sounds like which particular batch we're processing when we're doing multiple batches, maybe `idx_in_batch` is more appropriate here?

Reply:
Renamed to `idx_in_batch` in https://github.com/proteneer/timemachine/pull/1241/commits/3540ce7380cce947b5984fb02f7cd631a37c2306